_std/macros/genetics_traits.dm

```diff
@@ -1,2 +1,3 @@
 //for trait / genetics checks
 #define isalcoholresistant(x) ((x.bioHolder && x.bioHolder.HasEffect("resist_alcohol")) || (x.traitHolder && x.traitHolder.hasTrait("training_drinker")))
+#define isdwarf(x) ((x.bioHolder && x.bioHolder.HasEffect("dwarf")) || (x.traitHolder && x.traitHolder.hasTrait("dwarf")))
```

Comment:
```suggestion

```
If you're not using this, you don't need to add it.
Dwarfism also doesn't have mechanical effects (afaik), unlike the alcoholic trait.

Reply:
Yea I mostly just copied whatever the career alcoholic trait did seeing as that was functionally the same (and im not exactly the greatest coder). Its removed now though

Reply:
> If you're not using this, you don't need to add it. Dwarfism also doesn't have mechanical effects (afaik), unlike the alcoholic trait.

Dwarfism gives cold res right?